### ISSUE TITLE
Update NetCDF History information

### ISSUE THREAD

**Issue by Zeitsperre** [Ouranosinc/xclim#80]
One of the useful things that bash utilities like `ncks` and `cdo` do is they append comments to the history field of outputted netcdf files the processes that have been performed on the files. For example:
```
:Conventions = "CF-1.6" ;
		:history = "Sun Jul 29 07:19:00 2018: cdo -O -f nc sellonlatbox,-96,-72,39,52 /<servername>/smith/scripts/eccc_07132018/crcm5_data/bbw/psl/psl_bbw_199701_se.nc /<servername>/smith/scripts/eccc_07132018/cdo_subset/bbw/psl/psl_bbw_199701_se_ECCCsubset.nc\n"
```
It would be good to add a function that on exit or on write success adds a history entry that specifies:
* The date that the file was modified
* The `xclim` function performed
* The filepaths (I/O)

**Comment by Balinus** [comment]
I'd drop the file paths for security reasons.

**Comment by Zeitsperre** [comment]
That seems fair. Maybe just keep the file base names?

**Comment by Balinus** [comment]
Good idea!

**Comment by Zeitsperre** [comment]
@huard This seems to be addressed in #116. Close?
